bitnami/redis/README.md

```diff
@@ -132,6 +132,7 @@ The following table lists the configurable parameters of the Redis chart and the
 | `master.persistence.size`                     | Size of data volume                                                                                                                                 | `8Gi`                                                   |
 | `master.persistence.matchLabels`              | matchLabels persistent volume selector                                                                                                              | `{}`                                                    |
 | `master.persistence.matchExpressions`         | matchExpressions persistent volume selector                                                                                                         | `{}`                                                    |
+| `master.statefulset.extraEnv`                 | Additional environement variables passed to the pod of the master's stateful set set                                                                | `[]`
```

Comment:
I think we should place it under `master.extraEnvVars` just like we do with `master.extraFlags`, as this not a direct property of statefulset but the pod.

https://github.com/bitnami/charts/blob/4941fb52c1c90e2438ff7ef232332d06f4b159e5/bitnami/redis/values.yaml#L251-L267